crates/compiler/builtins/roc/Str.roc

```diff
@@ -47,7 +47,7 @@ interface Str
         withPrefix,
     ]
     imports [
-        Bool.{ Bool },
+        Bool.{ Bool, Eq },
```

Comment:
Is this import used? Do we have to import this whenever we use `==`?

Reply:
Nope!

Reply:
Oh oops, sorry, the answer is yes, because this is a builtin module - builtins have to explicitly enumerate their builtin imports. For user code, no.